easypost/services/base_service.py

```diff
@@ -30,7 +31,9 @@ def _snakecase_name(self, class_name: str) -> str:
     def _class_url(self, class_name: str) -> str:
         """Generate a URL based on class name."""
         transformed_class_name = self._snakecase_name(class_name)
-        if transformed_class_name[-1:] in ("s", "h"):
+        if transformed_class_name in _EXCLUDED_CLASS_NAMES:
+            return f"/{transformed_class_name}"
```

Comment:
What is this for? I don't quite understand why we are adding this

Reply:
This is an unfortunate hack needed to get around our "magic" class transformations. Without this change, it'll double pluralize the name to `ups_oauth_registrationses` which then won't work properly. So we basically have an override list that just takes this as is without transforming it.

Reply:
Ugh.